runtime/service_integ_test.go

```diff
@@ -351,71 +355,104 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 				TimeoutSeconds: 300,
 			}
 
-			resp, err := fcClient.CreateVM(ctx, req)
-			require.NoError(t, err, "failed to create vm")
+			resp, createVMErr := fcClient.CreateVM(ctx, req)
+			if createVMErr != nil {
+				matches, err := findProcess(ctx, findFirecracker)
+				if err != nil {
+					return err
+				}
+				return fmt.Errorf(
+					"failed to create a VM while there are %d Firecracker processes: %w",
+					len(matches),
+					createVMErr,
+				)
+			}
 
-			var containerWg sync.WaitGroup
+			eg := &errgroup.Group{}
 			for containerID := 0; containerID < int(containerCount); containerID++ {
-				containerWg.Add(1)
-				go func(containerID int) {
-					defer containerWg.Done()
-					testMultipleExecs(
+				containerID := containerID
```

Comment:
was this leftover from some previous code?

Reply:
It is for https://eli.thegreenplace.net/2019/go-internals-capturing-loop-variables-in-closures/, but I think it is unnecessary for this loop. Will remove.

Reply:
Hmm, seems golangci-lint (scopelint) is not happy about that. It could be a false positive though.